### DISCUSSION TITLE
Service firestore is not available

### DISCUSSION THREAD

**Discussion by avibhardwaj233** [firebase/firebase-js-sdk#5952] [Q&A] [answered]
I am trying to use firestore as my Database with Next.js but whenever I initialize it with `getFirestore()` then it shows me this error:
`Error: Service firestore is not available`

This error never happened before in any other framework. I also tried creating a new firebase project but it didn't work.
Any help will be greatly appreciated.

**Comment by avibhardwaj233** [comment, marked as answer]
The error is now actually solved, I was just importing `initializeApp()` in the wrong way.

**Reply by NomadicDeveloper22** [reply]
Having the same issue. In what way were you importing before? And what ended up being the correct way?
